pandas/tests/util/test_assert_numpy_array_equal.py

```diff
@@ -175,3 +177,18 @@ def test_numpy_array_equal_copy_flag(other_type, check_same):
             tm.assert_numpy_array_equal(a, other, check_same=check_same)
     else:
         tm.assert_numpy_array_equal(a, other, check_same=check_same)
+
+
+def test_numpy_array_equal_contains_na():
```

Comment:
Nice - can you also add a test using `nulls_fixture` that ensures equality passes?